src/install/strv.c

```diff
@@ -88,7 +88,7 @@ char **strv_copy(char *const *l)
         return r;
 }
 
-unsigned int strv_length(char *const *l)
+unsigned strv_length(char *const *l)
```

Comment:
I said separate commit. It doesn't make any sense to squash this into a commit titled "reduce cppcheck warnings".

Reply:
Can you please be more specific ? You want me to send a different PR for `unsigned int` changes or another commit in this same PR?

If you check my commit message, I have mentioned the list of changes included in the description. I get your point on commit title is a bit misleading, the other changes are cosmetic and nitpicking ones, so I piggybacked it in the same commit. Do you mind if I change the title to `fix(install): cosmetic and nitpicking changes`

Reply:
And I rebase and force push to my branch every time, so there are no multiple commits here to squash.

Reply:
> Can you please be more specific ? You want me to send a different PR for `unsigned int` changes or another commit in this same PR?

No, to add another commit with the `unsigned int` changes to this PR.

> the other changes are cosmetic and nitpicking ones, so I piggybacked it in the same commit.

Commits are cheap. There's no need to mix cosmetic changes with real changes.